lib/attrsets.nix

```diff
@@ -43,6 +44,16 @@ rec {
       [ { name = head attrPath; value = setAttrByPath (tail attrPath) value; } ];
 
 
+  /* Map function f over the value of the attribute described by attrPath.
+     For instance, mapAttrByPath [ "x" "y" ] (a: a+1) { x.y = 1; } returns
+     { x.y = 2; }. If the path doesn’t exist the default value is used. */
```

Comment:
Changed the implementation in the newest version, according to aszlig’s input.

Do you think we should rename that function? I can’t think of another word but map. But I will change the docstring to start with “Apply function …”.